lib/errors/InvalidIdentityAssetLockProofHeightError.js

```diff
@@ -0,0 +1,23 @@
+const ConsensusError = require('./ConsensusError');
+
+class InvalidIdentityAssetLockProofHeightError extends ConsensusError {
+  /**
+   *
+   * @param {number} height
+   */
+  constructor(height) {
```

Comment:
It would be nice to pass consensus core chain locked height as well and make error message more useful for users.

Reply:
Do we need a new method to get it? Or should we return it from verifyChainLockHeight?